packages/docs/pages/Switch/SwitchPage.tsx

```diff
@@ -9,21 +9,21 @@ const SwitchPage = () => (
     <H0>Switch</H0>
 
     <Text>
-      Switches are a stylized <Code>input[type="checkbox"]</Code> with controllable checked/unchecked states.
+      Switches are a stylized <Code>input[type="checkbox"]</Code> with controllable checked/unchecked states. Switches
+      are intended to be used for immediate toggle actions and are therefore not intended to be used in forms.
     </Text>
 
     <CodePreview>
       {/* jsx-to-string:start */}
       {function Example() {
         const [checked, setChecked] = useState(false);
         const handleChange = () => setChecked(!checked);
+        const boxColor = checked ? 'brand' : 'secondary20';
```

Comment:
I don't think the `brand` color on `checked` is very accessible 🤔 Could we just leave the normal background color?

Reply:
Which background color is normal? I was going to use `primary` but then the switch blends in.

Reply:
Just the inherit white background should be good.